Lib/ast.py

```diff
@@ -1097,33 +1166,19 @@ def visit_Name(self, node):
         self.write(node.id)
 
     def _write_docstring(self, node):
-        def esc_char(c):
-            if c in ("\n", "\t"):
-                # In the AST form, we don't know the author's intentation
-                # about how this should be displayed. We'll only escape
-                # \n and \t, because they are more likely to be unescaped
-                # in the source
-                return c
-            return c.encode('unicode_escape').decode('ascii')
-
         self.fill()
         if node.kind == "u":
             self.write("u")
-
-        value = node.value
-        if value:
-            # Preserve quotes in the docstring by escaping them
-            value = "".join(map(esc_char, value))
-            if value[-1] == '"':
-                value = value.replace('"', '\\"', -1)
-            value = value.replace('"""', '""\\"')
-
-        self.write(f'"""{value}"""')
+        value, quote_types = self._str_literal_helper(node.value, ('"""', "'''"))
+        quote_type = quote_types[0]
+        self.write(f"{quote_type}{value}{quote_type}")
```

Comment:
I feel like this is extremely similar to `_write_str_avoiding_backslashes`. Would you mind to use it here (and add a `**kwargs` parameter to pass there and pass it to helper)

Reply:
Thanks, changed!